### DISCUSSION TITLE
Channel was opened to me, but my node does not kow the channel?

### DISCUSSION THREAD

**Discussion by DerEwige** [ACINQ/eclair#2372] [Node operators]
From what I have gathered so fare another node tried to opened a channel to my node.
It kind of worked but actually did not really?

The channel shows up when using “eclair-cli channels” but does not show up under “eclair-cli usablebalance”

My node does not make any announcement for this channel when starting up.
When looking at the channel in lightning explorer my side is just empty: https://amboss.space/edge/823022936318017537 

How did this happen? How can I fix this? 
I can see the logs from the creation 2022-08-08 12:58 but normally 5 blocks later it should switch to enabled. But this did not happen the block was mined 2022-08-08 13:31 and I restarted my node exactly right after this block was mined by pure chance.

Could this somehow have lead to my node never enabeling the channel?

**Comment by t-bast** [comment]
This is perfectly normal, if someone opened a channel to you all the funds are initially on their side.
There is no money on your side, so the channel isn't enabled in the direction `you -> your peer`.
And it doesn't show up in `usablebalances` because you don't have any balance in this channel yet.
You need to wait for payments to be relayed to have some balance in this channel, it will enable itself automatically at that point.

**Reply by DerEwige** [reply]
Ok perfect.
I feared I broke something with my plugins, or a badly timed restart or some other shenanigans 

Thanks for clarifying

**Comment by DerEwige** [comment]
Iw as expecting the channel to show up under “eclair-cli usablebalance” with 100% can receive, and 0% can sen.

Because it did not show up at all. I had to change my active balancer.
To also look for remotely funded channels that are not enabled and included them as potential target for circular payments
